src/components/structures/auth/Registration.js

```diff
@@ -542,7 +546,7 @@ module.exports = createReactClass({
             return <div className="mx_AuthBody_spinner">
                 <Spinner />
             </div>;
-        } else {
+        } else if (this.state.flows.length) {
```

Comment:
Unsure how this helps not show the spinner (I assume the the one 2 lines above this line) ... ?

Reply:
lines 242 and 249 stop the spinner from showing, line 549 stops `RegistrationForm` being rendered with 0 flows

Reply:
Ahh, right, `!this.state.flows`, gotcha :+1: